src/core/server/integration_tests/saved_objects/routes/test_utils.ts

```diff
@@ -6,36 +6,7 @@
  * Side Public License, v 1.
  */
 
-import { executionContextServiceMock } from '@kbn/core-execution-context-server-mocks';
-import { ContextService } from '@kbn/core-http-context-server-internal';
-import { createHttpServer, createCoreContext } from '@kbn/core-http-server-mocks';
-import type { SavedObjectsType } from '@kbn/core-saved-objects-server';
-import { contextServiceMock, coreMock } from '../../../mocks';
-
-const defaultCoreId = Symbol('core');
-
-export const setupServer = async (coreId: symbol = defaultCoreId) => {
-  const coreContext = createCoreContext({ coreId });
-  const contextService = new ContextService(coreContext);
-
-  const server = createHttpServer(coreContext);
-  await server.preboot({ context: contextServiceMock.createPrebootContract() });
-  const httpSetup = await server.setup({
-    context: contextService.setup({ pluginDependencies: new Map() }),
-    executionContext: executionContextServiceMock.createInternalSetupContract(),
-  });
-  const handlerContext = coreMock.createRequestHandlerContext();
-
-  httpSetup.registerRouteHandlerContext<any, 'core'>(coreId, 'core', (ctx, req, res) => {
-    return handlerContext;
-  });
-
-  return {
-    server,
-    httpSetup,
-    handlerContext,
-  };
-};
+import { SavedObjectsType } from '@kbn/core-saved-objects-server';
 
 export const createExportableType = (name: string): SavedObjectsType => {
```

Comment:
Can we remove this as well?

Reply:
See https://github.com/elastic/kibana/pull/143483#discussion_r998194523. I'm moving as little code as is needed.